### PR TITLE
Update install guide

### PR DESCRIPTION
**Pull request by nadonomy**
Updated styling to present options clearer and with less scrolling. Removed extraneous copy & elements. Looks like:

Screenshot 2019-09-09 at 16 37 56

& if there's server configuration:

Screenshot 2019-09-09 at 16 37 20

Not merge ready. `configure_riot_button` needs to be dynamically set, using whatever configuration link we arrive at. Ideally this is either a universal link for both iOS & Android, or if not we detect user agent and inject appropriately.

### CODE REVIEW

**Comment by dbkr** [comment]
Made button update dynamically (I also made it a link instead as buttons can't have hrefs). Other changes look good.

File: src/vector/mobile_guide/index.js [hunk 1]
```diff
@@ -92,6 +92,9 @@ async function initPage() {
     if (isUrl && !isUrl.endsWith('/')) isUrl += '/';
 
     if (hsUrl !== 'https://matrix.org/') {
+        document.getElementById('configure_riot_button').href =
+            "https://riot.im/config/config?hs_url=" + encodeURIComponent(hsUrl)  +
```
Comment (turt2live) [inline src/vector/mobile_guide/index.js]: configconfig?

**Comment by aaronraimist** [comment]
"Back to Desktop"?

"Back to Riot.im" was already confusing for many users. I'm not really sure what back to desktop even means.

It would be really great to say something closer to "Use Riot in my web browser instead (not recommended)", obviously word that better if you can but back to desktop doesn't really make sense.

**Comment by nadonomy** [comment]
> "Back to Desktop"?
> 
> "Back to Riot.im" was already confusing for many users. I'm not really sure what back to desktop even means.
> 
> It would be really great to say something closer to "Use Riot in my web browser instead (not recommended)", obviously word that better if you can but back to desktop doesn't really make sense.

Viewed in a vacuum on desktop, I'd agree. However, if we empathise with the journey the user has taken to arrive at this context:

- They know the URL of a Riot instance, which means they're either (a) already a Riot web user or (b) someone shared a Riot URL with them, presumably with more context.
- They're viewing this on a mobile device (redirect by user agent)

Considering the most prevalent mobile browsers (Safari on iOS & Chrome on Android) use 'Request Desktop Site' I think it's wise to borrow affordance from their nomenclature, albeit dropping 'Site'.

**Comment by aaronraimist** [comment]
Back to Desktop Site would be a lot clearer imo

**Comment by aaronraimist** [comment]
Although I still don't understand why it is "Back to". They loaded riot.im in their browser and were immediately redirected to this page right? They were never on the desktop site to begin with.

**Comment by nadonomy** [comment]
> Although I still don't understand why it is "Back to". They loaded riot.im in their browser and were immediately redirected to this page right? They were never on the desktop site to begin with.

You're right— "Go to Desktop Site" could be more appropriate, depending on their entry path.

**Pull request merged**